aws-s3/scripts-ccao-data-raw-us-east-1/spatial-transit.R

```diff
@@ -78,14 +80,26 @@ walk(metra_feed_dates_list, get_metra_feed)
 ##### Pace #####
 pace_feed_dates_list <- c(
   "2015-10-16", "2016-10-15", "2017-10-16", "2018-10-17",
-  "2019-10-22", "2020-09-23", "2021-03-15"
+  "2019-10-22", "2020-09-23", "2021-03-15", "2023-09-24"
 )
 
 get_pace_feed <- function(feed_date) {
```

Comment:
I think the TransitFeeds DB is deprecated/offline now. Are we still getting all the data we need for this?

Reply:
Yep, these late 2023 dates and their data are present in the Athena tables.